tests/sentry/incidents/endpoints/test_serializers.py

```diff
@@ -435,7 +435,7 @@ def test_channel_timeout(self, mock_get_channel_id):
         )
 
     @patch(
-        "sentry.integrations.slack.utils.get_channel_id_with_timeout",
+        "sentry.integrations.slack.utils.channel.get_channel_id_with_timeout",
```

Comment:
@mgaeta I think this function gets re-exported by the `__init__.py`. No need to add `channel`.